winforms/src/toga_winforms/widgets/progressbar.py

```diff
@@ -4,43 +4,79 @@
 
 from .base import Widget
 
+# Implementation notes
+# ====================
+#
+# * Inactive Winforms ProgressBars have a distinct style. We use that style as an
+#   indicator for whether the ProgressBar is running.
```

Comment:
On Windows 7 I see the same as [this image](https://www.csharpstudy.com/WinForms/WinForms-progressbar.aspx): both Continous and Blocks styles appear continuous.

Reply:
Looks like I misunderstood what this is doing. "Blocks" style is no longer honoured by Winforms; in Vista, it was a rendering style discrete "chunks". There's now effectively no difference between Blocks and Continuous, other than an animation between values.

(I swear - OS visual designers seem to be on a continuous drive to *remove* useful distinctions in their UX...)

Reply:
I've modified this so that the Winforms progress bar doesn't use "Blocks" any more; it definitely wasn't the right approach for an "animating determinate" progress bar. This means we need to introduce a proxy variable to track "running", because we need to move an non-running indeterminate progress bar to "Continuous" to stop the animation.